app.go

```diff
@@ -120,10 +120,16 @@ type OIDCConfig struct {
 }
 
 type DERPConfig struct {
-	URLs            []url.URL
-	Paths           []string
-	AutoUpdate      bool
-	UpdateFrequency time.Duration
+	ServerEnabled    bool
+	ServerRegionID   int
+	ServerRegionCode string
+	ServerRegionName string
+	STUNEnabled      bool
+	STUNAddr         string
```

Comment:
Can we split this into `DERPServerConfig` and `DERPSTUNConfig` ?

At some point I want to have a go at getting rid of all the manual reading of options from viper, I think it can do that...

Reply:
maybe look into using HCL struct based parsing?